README.md

```diff
@@ -5,7 +5,7 @@
 Project requirements can be found [here](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md)
 
 ## Kanban Board
-We are two members in team 
+We are two members in the team 
```

Comment:
- [ ] Kindly add to your instructions the db commands that your users will require to run your project:  `rails db:create, rails db:migrate, rails db:seed`.  That way, your users will run your project smoothly.